docs/resources/monitor.md

```diff
@@ -67,6 +67,7 @@ For example, if the value is set to `300` (5min), the `timeframe` is set to `las
 - `no_data_timeframe` (Number) The number of minutes before a monitor will notify when data stops reporting. Provider defaults to 10 minutes.
 
 We recommend at least 2x the monitor timeframe for metric alerts or 2 minutes for service checks.
+- `notification_preset_name` (String) Toggles the display of additional content sent in the monitor notification. Valid values are: `show_all`, `hide_query`, `hide_handles`, and `hide_all`. Valid values are `show_all`, `hide_query`, `hide_handles`, `hide_all`.
```

Comment:
The enum already tells tfplugindocs about valid values, so we should remove the valid values from the description to avoid the duplication